spec/system/task_spec.rb

```diff
@@ -58,57 +58,53 @@
   end
 
   describe 'Task編集' do
+    
+    let(:task_done){ create(:task, :done, project: project) }
```

Comment:
associationの定義により、この部分もprojectの記載が不要になると思います。
他のletについても確認してみてください。
```suggestion
    let(:task_done){ create(:task, :done) }
```

Reply:
7281992
上記修正しましt